GeometryDashAPI/Data/GameData.cs

```diff
@@ -28,20 +28,21 @@ public virtual async Task LoadAsync(string fileName)
             if (!File.Exists(fileName))
                 throw new FileNotFoundException($"file does not exists: '{fileName}'");
 
-            // File > Bytes > XOR > ToString > Replace > Base64 > Gzip
 #if NETSTANDARD2_1
-            var data = await File.ReadAllBytesAsync(fileName);
+            await using var file = new FileStream(fileName, FileMode.Open, FileAccess.Read, FileShare.Read, 4096, useAsync: true);
 #else
             using var file = new FileStream(fileName, FileMode.Open, FileAccess.Read, FileShare.Read, 4096, useAsync: true);
-            var data = new byte[file.Length];
-            var read = await file.ReadAsync(data, 0, data.Length);
 #endif
-            var dataZip = Encoding.ASCII.GetString(Crypt.XOR(data, 0xB)).Split('\0')[0];
-            var resultPlist = Crypt.GZipDecompress(GameConvert.FromBase64(dataZip));
+            var data = new byte[file.Length];
+            await file.ReadAsync(data, 0, data.Length);
 
-            DataPlist = new Plist(Encoding.ASCII.GetBytes(resultPlist));
-        }
+            var xor = Crypt.XOR(data, 0xB);
+            var index = xor.GetIndexOfNullByte();
```

Comment:
It can be implemented easier and will be even faster
```cs
xor.AsSpan().IndexOf((byte)0)
```

Because inside `IndexOf` use a simd instructions.

Proof:

```cs
using System;
using System.Linq;
using BenchmarkDotNet.Attributes;

namespace GeometryDashAPI.Benchmarks.Benchmarks;

[MemoryDiagnoser]
public class IndexOfBenchmark
{
    private static readonly byte[] data = Enumerable.Range(0, 4096)
        .Select(x => (byte)(1 + (byte)(x % 250)))
        .Concat(new byte[] { 0 })
        .ToArray();

    [Benchmark]
    public int IndexOf() => data.AsSpan().IndexOf((byte)0);

    [Benchmark]
    public int GetIndexOfNullByte() => data.GetIndexOfNullByte();
}
```

|             Method |        Mean |     Error |    StdDev | Allocated |
|------------------- |------------:|----------:|----------:|----------:|
|            IndexOf |    84.86 ns |  1.939 ns |  5.657 ns |         - |
| GetIndexOfNullByte | 2,244.83 ns | 38.261 ns | 35.789 ns |         - |

Reply:
oh..